src/controller/login.controller.ts

```diff
@@ -254,4 +254,38 @@ export class LoginController {
       await this.fileService.writeTempFile(this.tempFile);
     }
   }
+
+  /**
+   * Returns Promise which resolves to true/false if the user is currently logged in
+   * @returns {Promise<boolean>} - isUserLoggedIn
+   * @memberof LoginController
+   */
+  public async isUserLoggedIn(): Promise<boolean> {
+    let activeAccountId: string;
+    try {
+      this.tempFile = await this.fileService.getTempFile();
+      if (!this.tempFile) {
+        return false;
+      }
+
+      activeAccountId = Object.keys(this.tempFile).find(
+        (e) => this.tempFile[e].active,
+      ) as string;
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.log(error);
```

Comment:
Do you want to add some Message to it or just log it plain like that ?

Reply:
Not to sure how this will behave for an error, god chance it will print `[object]`